<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiAxisConfigDialog</class>
 <widget class="QDialog" name="uiAxisConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,1,1,1,10,1" columnstretch="1,3,3,1,1,1,1,0">
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X1轴功能</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="x1">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>失效</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>使能</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>X2轴功能</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="x2">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>失效</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>使能</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>X轴最小值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>R1轴功能</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="r1">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>失效</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>使能</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>R轴最大值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>R2轴功能</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="r2">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>失效</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>使能</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>R轴最小值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Z1轴功能</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="z1">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>失效</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>使能</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Z轴最大值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Z2轴功能</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="z2">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>失效</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>使能</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Z轴最小值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="6">
      <widget class="QLabel" name="type_show">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/images/gauge_1</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="3">
      <widget class="QLineEdit" name="z_min"/>
     </item>
     <item row="5" column="3" colspan="3">
      <widget class="QLineEdit" name="z_max"/>
     </item>
     <item row="4" column="3" colspan="3">
      <widget class="QLineEdit" name="r_min"/>
     </item>
     <item row="3" column="3" colspan="3">
      <widget class="QLineEdit" name="r_max"/>
     </item>
     <item row="2" column="3" colspan="3">
      <widget class="QLineEdit" name="x_min"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>X轴最大值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="8">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>轴配置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="3">
      <widget class="QLineEdit" name="x_max"/>
     </item>
     <item row="1" column="6" colspan="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6" colspan="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6" colspan="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6" colspan="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6" colspan="2">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>后挡料类型</string>
       </property>
      </widget>
     </item>
     <item row="9" column="6" colspan="2">
      <widget class="QPushButton" name="sure">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="type">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>类型1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>类型2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>类型3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>类型4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="3" colspan="3">
      <widget class="QLineEdit" name="bendRate"/>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>折弯系数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
